README.md

```diff
@@ -56,13 +56,12 @@ metadata:
   name: read-configmaps
```

Comment:
this role no longer just `read-configmaps`. Maybe rename it?

Reply:
I was able to use existed roles when I removed `leader election` see https://github.com/che-incubator/configbump/pull/8#discussion_r466983219